app/src/main/java/vn/mbm/phimp/me/Settings.java

```diff
@@ -267,8 +268,15 @@ public void onClick(DialogInterface dialog, int which)
 										}*/
 										
 									}
-									getActivity().sendBroadcast(new Intent(Intent.ACTION_MEDIA_MOUNTED, Uri.parse("file://"
-								            + Environment.getExternalStorageDirectory()))); 
+                                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
There are shortcuts in IDE which you can use for this. 
My suggestion is to use shortcut on every file you work :)